src/runtime/validator.ts

```diff
@@ -43,16 +45,10 @@ export const useChecker = (
     const formattedResult = formatter?.(results)
     const message = [
       `HTML validation errors found for ${chalk.bold(url)}`,
-      formattedResult
+      ...(formattedResult ? [formattedResult] : [])
     ].join('\n')
 
-    if (valid) {
-      if (logLevel === 'verbose' || logLevel === 'warning') {
-        console.warn(message)
-      }
-    } else {
-      console.error(message)
-    }
```

Comment:
shouldn't we still log warnings in this case?

Reply:
Which cases do we have?
- `valid && !results.length` -> all good
- `valid && results.length` -> warning
- `!valid && results.length` -> error

Is that right? I might be a little confused about the possibilities.

Reply:
yes - first case we should never get this far - you fixed it

but we need to handle both second and third cases separately, right?

Reply:
You're totally right, I've reverted that case and added tests for it. It was a misunderstanding on my site. I was confused by the cases named above.